README.md

```diff
@@ -91,668 +65,12 @@ platforms have different sub-maintainers, please refer to the file
-
-#### 4.4.3 Build the images and files
-For the 96boards/cannes:
-```
-$ make PLATFORM=stm-b2260
-```
-For the legacy cannes family:
-```
-$ make PLATFORM=stm-cannes
-```
-
-#### 4.4.4 Prepare, install and run the software
-For 96board/cannes:
-Copy the generated 'optee.bin' on target SD or USB stick and insure the boot
-media boot scripts defines optee and non-secure worlds boot.
-
-More info to come...
-
----
-### 4.5 Allwinner A80
-
-#### 4.5.1 Locked versus unlocked A80 boards
-**Important!** All A80 boards sold to the general public are boards where secure
-side has been locked down, which means that you **cannot** use them for secure
-side development, i.e, it will not be possible to put OP-TEE on those devices.
-If you want to use A80 board for secure side development, then you will need to
-talk to
-[Allwinner](https://github.com/OP-TEE/optee_os/blob/master/MAINTAINERS.md)
-directly and ask if it is possible get a device from them.
-
-#### 4.5.2 Get the compiler and source
-Follow the instructions in the "4.2 Basic setup".
-
-#### 4.5.3 Build
-```
-$ cd optee_os
-$ export PLATFORM=sunxi
-$ export CROSS_COMPILE=arm-linux-gnueabihf-
-$ make
-```
-
-#### 4.5.4 Prepare the images to run on A80 Board
-
-Download Allwinner A80 platform SDK, the SDK refers to Allwinner A80 platform
-SDK root directory. A80 SDK directory tree looks like this:
-```
-SDK/
-    Android
-    lichee
-```
-`Android` contains all source code related to Android and `lichee`
-contains the bootloader and Linux kernel.
-
-##### 4.5.4.1 Copy OP-TEE output to package directory
-Copy the OP-TEE output binary to `SDK/lichee/tools/pack/sun9i/bin`
-
-```
-$ cd optee_os
-$ cp ./out/arm32-plat-sunxi/core/tee.bin SDK/lichee/tools/pack/sun9i/bin
-```
-
-##### 4.5.4.2 Build Linux kernel
-In the `lichee` directory, run the following commands:
-```
-$ cd SDK/lichee
-$ ./build.sh
-```
-
-##### 4.5.4.3 Build Android
-In the Android directory, run the following commands:
-```
-$ cd SDK/android
-$ extract-bsp
-$ make -j
-```
-
-##### 4.5.4.4 Create the Android image
-In the Android directory, run the following commands:
-```
-$ cd SDK/android
-$ pack
-```
-The output image will been signed internally when packed. The output image name
-is `a80_android_board.img`.
-
-##### 4.5.4.5 Download the Android image
-Use `Allwinner PhoenixSuit` tool to download to A80 board.
-Choose the output image(`a80_android_board.img`), select download and wait
-for the download to complete.
-
-#### 4.5.5 Boot and run the software on A80 Board
-When the host platform is Windows, use a console application to connect A80
-board `uart0`. In the console window, You can install OP-TEE linux kernel
-driver `optee.ko`, load OP-TEE-Client daemon `tee-supplicant` and run
-the example "hello world" Trusted Application, do this by running:
-```
-$ insmod /system/vendor/modules/optee.ko
-$ /system/bin/tee-supplicant &
-$ /system/bin/tee-helloworld
-```
-
----
-### 4.6 Freescale MX6UL EVK
```

Comment:
@MrVan , yes, but there is no hurry doing that I think. What I have in the pipe is this change and most of the information removed here went into https://github.com/OP-TEE/build/pull/94. So I was thinking if it was possible in your case to:
a) Add the needed xml-files (manifests) in https://github.com/OP-TEE/manifest
b) If it makes sense, create similar "helper" Makefile as we have in https://github.com/OP-TEE/build. If it doesn't make sense, then we can create a separate file for the device(s) your maintaining in like I've done with all the "Add __xyz__.md" file in the PR (https://github.com/OP-TEE/build/pull/94).

Again, no hurry, since I don't think the updated doc patches will be part of the 2.3.0 release (maybe, maybe not).